src/lib/components/VersionField.js

```diff
@@ -28,7 +28,7 @@ export class VersionField extends Component {
             {' '}
             semver.org
           </a>
-          , but any version string is accepted.
+          ) but any version string is accepted.
```

Comment:
```suggestion
          ) but any string is accepted.
```

Also, I suggest to remove "version string" to just use "string" here, because it better suggests that any string can be used as a version (while "version string" may suggest `x.y.z`). E.g. `revision-conference-80` is accepted in this field, as well as

```
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

without triggering any validation error